### ISSUE TITLE
Special characters of the german language

### ISSUE THREAD

**Issue by solaris1967** [servocoder/RichFilemanager#188]
Hello,

in german language, special characters are normal. The special characters are for example the letters: ä, ö, and ü.

In order to replace these file signs when uploading the files, I added the following code in the file "/connectors/php/BaseFilemanager.php" in line 512:

$replacements = [' '=>'_', '\''=>'_', '/'=>'', '\\'=>'', 'ä'=>'ae', 'ö'=>'oe', 'ü'=>'ue', 'Ä'=>'Ae', 'Ö'=>'Oe', 'Ü'=>'Ue', 'ß'=>'ss'];

The file must then always be saved in UTF8 format!

Greetings,

Frank from Germany

**Comment by servocoder** [comment]
You could set PHP `charsLatinOnly` configuration option to `false`. In this case it should not clean up non-latin chars. I believe this would be a solutions for your case.

**Comment by solaris1967** [comment]
Yes that's right. But unfortunately this extension is not installed with every server. If the option `charsLatinOnly ` is set to `FALSE`, the file is uploaded but can not be displayed anymore. If I set the option to `TRUE`, the manager will refuse the upload.
With my method, the filename is cleaned up by the German special characters and the user does not need to rename the file to his computer.

**Comment by servocoder** [comment]
Could you provide a filename that is not displayed after uploading? When `charsLatinOnly` option set to `false` it should not perform any chars replacement, so I wonder why it's not displayed for you.

**Comment by solaris1967** [comment]
All back. I found my mistake.
I have tried the XAMPP server under Windows 10. Since I had the problem.
With my private Debian server, which I also normal for the programming to use, everything works perfectly.
It must therefore be due to the Windows that the special characters are not encoded properly.

I also have to say that your program is very good and I use it very much! Thanks a lot for this!!!

Greetz from Germany
Frank
